### PR TITLE
(WSR 8) <Page/> - Remove upgrade prop

### PR DESCRIPTION
**Pull request by EgoziE**
- Props:
  - removed `upgrade` - component is now upgraded by default

### CODE REVIEW

File: src/Page/docs/PageTestStoriesDeprecated.js [hunk 1]
```diff
@@ -1,102 +0,0 @@
-import React from 'react';
```
Comment (simast) [inline src/Page/docs/PageTestStoriesDeprecated.js]: we should also remove this from `stories/e2e.js`:
https://github.com/wix/wix-style-react/blob/c73627a8765d2453a8ca6b63855cb20b31be78e8/stories/e2e.js#L8

File: src/Page/Page.js [hunk 2]
```diff
@@ -542,8 +542,6 @@ Page.propTypes = {
 
   /** z-index of the Page */
   zIndex: PropTypes.number,
-  /** When true the page will use height: 100% and not require a parent of `display: flex;flex-flow: column;`. Also Page.Content's may grow using `min-height: inherit`. Supports Page.Sticky. New header minimization approach.*/
-  upgrade: PropTypes.bool, // This Upgrade prop is only for documentation, the actual use is in index.js
```
Comment (simast) [inline src/Page/Page.js]: let's also clean up another `upgrade` prop passed to PageHeader in this file:
https://github.com/wix/wix-style-react/blob/c73627a8765d2453a8ca6b63855cb20b31be78e8/src/Page/Page.js#L299

File: src/PageHeader/PageHeader.scss [hunk 3]
```diff
@@ -90,10 +90,7 @@ See: https://css-tricks.com/flexbox-truncated-text/#article-header-id-3 */
     height: 30px;
     margin-top: 6px;
     margin-left: auto;
-
-    &.animationEnabled {
-      transition: margin 0.2s cubic-bezier(0.4, 0, 0.2, 1);
-    }
+    transition: margin 0.2s cubic-bezier(0.4, 0, 0.2, 1);
```
Comment (simast) [inline src/PageHeader/PageHeader.scss]: Hmm, we were only applying this animation for the old (non upgraded) Page before. Are we enabling this for the new page as well?

Comment (EgoziE) [reply src/PageHeader/PageHeader.scss]: I asked UX, disabling animation for now, we might add this in the future

**Pull request merged**